### PR TITLE
Condenses down the prison contraband spawner.

### PR DESCRIPTION
**Pull request by sergeirocks100**
## About The Pull Request

This removes all of the singular cigarettes and the Uplift Smooth pack from the prison contraband loot spawner.

This also removes the matchbox and the regular handcuffs from the spawner as well.

## Why It's Good For The Game

I play shiftstart prisoner a lot, and often, all i get when looking for contraband are oodles of lone cigarettes and nothing useful.

With less cigarettes clogging up the spawning pool, there will be more room for actually useful objects, such as radios and shivs, to spawn.

## Changelog
:cl:
balance: The prison contraband loot spawner has had its singular cigarettes, matchbox, and handcuffs removed to improve the chances of more useful items spawning.
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
Are regular handcuffs not a useful item?

**Comment by sergeirocks100** [comment]
There's not much of a point to them when zipties spawn right alongside them.

Same goes for matchboxes when there's lighters available.

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -81,23 +81,14 @@
...
 /obj/effect/spawner/lootdrop/prison_contraband
 	name = "prison contraband loot spawner"
-	loot = list(/obj/item/clothing/mask/cigarette/space_cigarette = 4,
-				/obj/item/clothing/mask/cigarette/robust = 2,
-				/obj/item/clothing/mask/cigarette/carp = 3,
-				/obj/item/clothing/mask/cigarette/uplift = 2,
-				/obj/item/clothing/mask/cigarette/dromedary = 3,
-				/obj/item/clothing/mask/cigarette/robustgold = 1,
-				/obj/item/storage/fancy/cigarettes/cigpack_uplift = 3,
-				/obj/item/storage/fancy/cigarettes = 3,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/lootdrop.dm:91]: would it be possible to turn these into their own loot pool, which you can then add as an entry in this loot list?

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 2]
```diff
@@ -81,23 +81,14 @@
...
+	loot = list(/obj/item/storage/fancy/cigarettes = 3,
 				/obj/item/clothing/mask/cigarette/rollie/cannabis = 4,
 				/obj/item/toy/crayon/spraycan = 2,
 				/obj/item/crowbar = 1,
 				/obj/item/assembly/flash/handheld = 1,
 				/obj/item/restraints/handcuffs/cable/zipties = 1,
-				/obj/item/restraints/handcuffs = 1,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/lootdrop.dm:97]: the same goes for these cable ties and handcuffs, I guess

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 3]
```diff
@@ -81,23 +81,14 @@
...
 				/obj/item/radio/off = 1,
 				/obj/item/lighter = 3,
-				/obj/item/storage/box/matches = 3,
 				/obj/item/reagent_containers/syringe/contraband/space_drugs = 1,
 				/obj/item/reagent_containers/syringe/contraband/krokodil = 1,
 				/obj/item/reagent_containers/syringe/contraband/crank = 1,
```
Comment (ATH1909) [inline code/game/objects/effects/spawners/lootdrop.dm:94]: and droog syringes

Comment (ATH1909) [reply code/game/objects/effects/spawners/lootdrop.dm:94]: etc.

**Comment by Mothblocks** [comment]
Why is having "more useful items" good for the game? Prisoner is not meant to be a role where you escape every time, it's a roleplay one.

**Comment by sergeirocks100** [comment]
Usually, the first thing i do when i spawn in as a prisoner is look through the permabrig to see what the contraband has to offer for me.

I usually hope to find things like radios, wallets, PDAs, and shivs for cutting up wood and bedsheets.

Radios and PDAs are good for allowing me to have some level of interaction with the outside of the station, and making me feel more like i'm in a bubble away from the chaos of the station rather than trapped in a metal box.

There have been quite a few times where i found none of these things, and instead found oodles of cigarettes that i would never end up smoking.

Having these items be buried under mountains of cancer sticks wouldn't make them much more valuable, as most wardens and other security personnel, in my experience, have been more than happy to give me these items, sometimes without me even asking first, and it just makes things worse on lowpop shifts where there is nobody in security to look after you.

I don't see how any of these items would make escape any easier except for possibly the radio and PDA, and even then, you could simply beg for someone to break you out of the permabrig through the newscaster or holopad.

Besides, i, and i assume most other people, don't play shiftstart prisoner so they can escape the permabrig, if they didn't want to be in the permabrig, they would not be shiftstart prisoners.

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 1 above)
Comment (coiax) [reply code/game/objects/effects/spawners/lootdrop.dm:91]: Yeah, this is a good idea.

**Review by coiax — CHANGES_REQUESTED** [review]
As ATH1909 suggests, you can bundle together the spawns, so you can have variety without weighing out other items.

**Pull request closed without merging**